src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
+  const smallerDiscount = 20;
+  const biggerDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return rentalCost - smallerDiscount;
+  } else if (days >= 7) {
+    return rentalCost - biggerDiscount;
+  }
```

Comment:
```suggestion
  } 
  
  if (days >= 7) {
    return rentalCost - biggerDiscount;
  }
```
it's better to avoid `else` and `else if` when it's possible (e.g. you return smth inside if block, as you do it here) - FYI